README.md

```diff
@@ -29,6 +29,19 @@ Update `prometheus/prometheus.yml` with the targets (server you wish to monitor)
 ./start-all.sh
 ```
 
+### Load original data to prometheus server
```

Comment:
Yes, it's same. I loaded the prometheus data which got from SCT performance test, grafana can present it correctly. It's helpful.

Data source:
1. Download from docker prometheus server, reference:
https://github.com/scylladb/scylla/wiki/How-to-report-a-Scylla-problem#prometheus
2. Get from Scylla-Cluster-Test log.
3. etc